farm-staking/farm-staking/src/token_attributes.rs

```diff
@@ -130,6 +134,30 @@ impl<M: ManagedTypeApi> Mergeable<M> for StakingFarmTokenAttributes<M> {
 }
 
 #[derive(TypeAbi, TopEncode, TopDecode, PartialEq, Debug)]
-pub struct UnbondSftAttributes {
-    pub unlock_epoch: u64,
+pub struct UnbondSftAttributes<M: ManagedTypeApi> {
```

Comment:
Isn't this live on mainnet? Should we do a custom decode function here?:-?

Reply:
Nope, I've created a new token for it.

Reply:
I'm asking because I see the version of `UnbondSftAttributes` with only 1 variable (`unlock_epoch`) is there from the first version of the contract.

Reply:
Fixed here: https://github.com/multiversx/mx-exchange-sc/pull/861

Didn't want to merge all branches again.

Reply:
Fixed here https://github.com/multiversx/mx-exchange-sc/pull/861